docs/AboutUs.md

```diff
@@ -43,17 +43,17 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 <img src="images/johndoe.png" width="200px">
 
 [[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[portfolio](team/lzher385.md)]
```

Comment:
It appears that this file is causing merge conflicts that are beyond limits to resolving on Github directly. Perhaps you could update this file with the current latest version from the master branch and commit, after which I can merge them.